spec/models/supplejack_api/record_metric_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+RSpec.describe SupplejackApi::RecordMetric do
+  describe '#attributes' do
+    let(:record_metric) { create(:record_metric, content_partner: ['NDHA']) }
+
+    it 'has a date' do
+      expect(record_metric.date).to eq Time.zone.today
+    end
+
+    it 'has a record_id' do
+      expect(record_metric.record_id).to eq 1
+    end
+
```

Comment:
Aren't you just testing the factory here?

Reply:
Yeah I'm just testing that the attributes can be set really :)